pipeline_runner/runner.py

```diff
@@ -6,6 +6,7 @@
 
 import docker  # type: ignore[import-untyped]
 from docker.models.networks import Network  # type: ignore[import-untyped]
+from requests import HTTPError
```

Comment:
To work with the other change, we need to change this import. The order is important for the linter to pass.
```suggestion
from docker.errors import APIError  # type: ignore[import-untyped]
from docker.models.networks import Network  # type: ignore[import-untyped]
```

You will also need to add
```python
from http import HTTPStatus
```
between these two lines
```python
from abc import ABC, abstractmethod
from time import time as ts
```

Reply:
Changes made!